src/main/scala/com.gu.gibbons/utils/UrlGenerator.scala

```diff
@@ -16,12 +17,17 @@ abstract class HashGenerator(md: MessageDigest) {
 }
 
 class UrlGenerator(settings: Settings, md: MessageDigest) extends HashGenerator(md) {
-  private def url(action: String, key: Key) =
-    s"${settings.bonoboUrl}/user/${key.consumerId}/${action}?${params(key, settings.salt)}"
+  private def keyUrl(action: String, key: Key) =
+    s"${settings.bonoboUrl}/user/${key.consumerId}/${action}?${keyParams(key, settings.salt)}"
```

Comment:
Just wondering if `/user/` should be `/key/` here?

Reply:
It's weird but it's because the routes `key/id/action` are for direct key manipulation in Bonobo while `user/id/action' are for deleting/extending keys by users

Reply:
We could change that but it needs to be changed in Bonobo as well @JustinPinner